### PR TITLE
Ft inventory

### PR DESCRIPTION
**Pull request by mjedwards**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist
Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+              className='pure-retail-logo'
+            />
+          </Col>
+
+          <Col className='gutter-row' span={4}>
+            <div style={style}>
+              <NavLink to='dashboard' activeClassName='activeLink'>
+                <p>Home</p>
+              </NavLink>
+              {/* <a href='#'>Home</a> */}
```
Comment (erferguson) [inline src/components/Navbar.js]: can probably be removed?

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            </div>
+          </Col>
+          <Col className='gutter-row' span={4}>
+            <div style={style}>
+              <NavLink to='inventory' activeClassName='activeLink'>
+                <p>Store</p>
+              </NavLink>
+              {/* <a href='#'>Store</a> */}
```
Comment (erferguson) [inline src/components/Navbar.js]: can probably be removed?

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            </div>
+          </Col>
+          <Col className='gutter-row' span={4}>
+            <div style={style}>
+              <NavLink to='account' activeClassName='activeLink'>
+                <p>Account</p>
+              </NavLink>
+              {/* <a href='#'>Account</a> */}
```
Comment (erferguson) [inline src/components/Navbar.js]: can be removed?

File: src/components/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+            </div>
+          </Col>
+          <Col className='gutter-row' span={4}>
+            <div style={style}>
+              <NavLink to='Profile' activeClassName='activeLink'>
+                <p>Profile</p>
+              </NavLink>
+              {/* <a href='#'>Profile</a> */}
```
Comment (erferguson) [inline src/components/Navbar.js]: can be removed?

File: src/components/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+          </Col>
+          <Col>
+            <a href={url} target='_blank'>
+              <Button className='nav-secondary'>View Store</Button>
+            </a>
+          </Col>
+        </Row>
+        <img src={storeLogo} alt='Store Logo' className='nav-store-logo' />
+      </Row>
+      {/* <Divider
```
Comment (erferguson) [inline src/components/Navbar.js]: 97-101, can be removed?

File: src/components/Orders/Orders.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
```
Comment (erferguson) [inline src/components/Orders/Orders.js]: doesn't appear that useSelector is being used in the code base

File: src/components/Orders/Orders.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+import * as creators from "../../state/actionCreators";
+import { Table, Tag, Button } from "antd";
```
Comment (erferguson) [inline src/components/Orders/Orders.js]: Tag and Button not called in code

File: src/components/Orders/Orders.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+import { connect } from "react-redux";
```
Comment (erferguson) [inline src/components/Orders/Orders.js]: connect not called in code, remove?

File: src/components/SellerAccount/SellerAccount.js [hunk 9]
```diff
@@ -1,55 +1,63 @@
-import React, { useState, useEffect } from 'react';
-import { Card, Button } from 'antd';
-import axios from 'axios';
-import Nav from '../elements/nav';
-import withAuth from '../Auth/axiosWithAuth';
+import React, { useState, useEffect } from "react";
+import { Card, Button } from "antd";
+import axios from "axios";
+// import Nav from '../elements/nav';
```
Comment (erferguson) [inline src/components/SellerAccount/SellerAccount.js]: remove if not using

**Review by erferguson — CHANGES_REQUESTED** [review]
just minor edits re: removing commented out code. good work, Mike!

**Comment by mjedwards** [comment]
made the suggested corrections

**Review by Cireimu — CHANGES_REQUESTED** [review]
Personally cannot approve this right now. It seems to be just a quick thing of flexbox or something but items just float off the screen, also have a ton of horizontal and the sizes vary drastically. Also, images should have standard sizing with max-width, upon loading this page I had two items that took up the entire screen. I can work with you on these things if needed but I need this to be fixed first before I can merge them.  
inventory screen

**Comment by mjedwards** [comment]
Everything should be fixed now.

**Pull request merged**